app/assets/javascripts/discourse/app/routes/new-message.js

```diff
@@ -56,6 +56,7 @@ export default DiscourseRoute.extend({
           e.send("createNewMessageViaParams", {
             topicTitle: params.title,
             topicBody: params.body,
+            recipients: "",
```

Comment:
This doesn't seem necessar, `recipients` defaults to  an empty array in the action.

Reply:
I found this was actually a problem in composer-user-selector, it was trying to split the `recipients` variable loaded in which did not work for the default empty array (everywhere else we use `createNewMessageViaParams` we specify recipients)